spec/components/candidate_interface/referees_review_component_spec.rb

```diff
@@ -94,6 +94,28 @@
       expect(change_email).to eq("Change email address for #{first_referee.name}")
       expect(change_relationship).to eq("Change relationship for #{first_referee.name}")
     end
+
+    context 'When referee type feature flag is on' do
```

Comment:
Do we want to test for when the feature flag is off?

Reply:
Added in 0d851b4